react/Icons/Success.js

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { calcIconSize, baseClassname } from './utils'
+
+const iconBase = {
+  width: 42,
+  height: 42,
+}
+
+class SuccessIcon extends PureComponent {
```

Comment:
Shouldn't this be pulled from the Styleguide?

Reply:
Yes! I didn't see that it was there already... Sorry about that hahaha